latest/Dockerfile

```diff
@@ -43,8 +43,20 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     # Clear apt caches to reduce image size
     rm -rf /var/lib/apt/lists/*
 
-# Get required python packages
-RUN pip install pyserial dependency-injector
+# Upgrade pip to avoid running out of memory
```

Comment:
Later versions of pip don't with python 2, which is what our ROS environment currently runs on. We need to make sure we install the correct version of pip so that our Dockerfile can build properly.

Reply:
that doesn't sound right, are you certain? I'm fairly certain that each install of pip is tied to a particular install of python

Reply:
it now manages to install torchvision but can't find the package detecto

Reply:
Yeah that was the error we got when we tried with an upgraded pip version